graphics/src/Image.cc

```diff
@@ -137,6 +137,19 @@ void Image::SavePNG(const std::string &_filename)
   FreeImage_Save(FIF_PNG, this->dataPtr->bitmap, _filename.c_str(), 0);
 }
 
+//////////////////////////////////////////////////
+void Image::SavePNGToBuffer(std::vector<unsigned char>& buffer)
+{
+  FIMEMORY *hmem = FreeImage_OpenMemory();
+  FreeImage_SaveToMemory(FIF_PNG, this->dataPtr->bitmap, hmem);
+  unsigned char* mem_buffer = nullptr;
+  unsigned int size_in_bytes = 0;
```

Comment:
use camelCase?

Reply:
Fixed in bc1a0a6256ff194baf94630dd18929e457a49a6f